### PR TITLE
T4267: Enable  signature button in additonal NS

### PR DESCRIPTION
**Pull request by Pix1234**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -951,6 +951,9 @@
 	'wmgUseEventLogging' => [
 		'default' => false,
 	],
+	'$wgExtraSignatureNamespaces' => [
+		'nonciclopediawiki' => [NS_HELP, NS_PROJECT],
```
Comment (paladox) [inline LocalSettings.php]: needs to be [ NS_HELP, NS_PROJECT ],

Also need to set a default 'default' => [],

File: LocalSettings.php [hunk 2]
```diff
@@ -951,6 +951,9 @@
 	'wmgUseEventLogging' => [
 		'default' => false,
 	],
+	'$wgExtraSignatureNamespaces' => [
```
Comment (paladox) [inline LocalSettings.php]: Needs to be wgExtraSignatureNamespaces minus the $

File: LocalSettings.php [hunk 3]
```diff
@@ -951,6 +951,10 @@
 	'wmgUseEventLogging' => [
 		'default' => false,
 	],
+	'wgExtraSignatureNamespaces' => [
+		'default' => [],
+		'nonciclopediawiki' => [ NS_HELP, NS_PROJECT],
```
Comment (paladox) [inline LocalSettings.php]: Forgot a space at the end of NS_PROJECT

Comment (JohnFLewis) [reply LocalSettings.php]: they should be new lines, not same line

File: LocalSettings.php [hunk 4]
```diff
@@ -2656,6 +2656,11 @@
 		'default' => [ 'png' ],
 	],
 	// Namespaces [MWNamespaces]
+		'wgExtraSignatureNamespaces' => [
+		'default' => [],
+		'nonciclopediawiki' => [ NS_HELP, 
+					NS_PROJECT ],
```
Comment (The-Voidwalker) [inline LocalSettings.php]: By on new lines, that means it should look rather more like
```
[
        NS_HELP,
        NS_PROJECT
],
```

File: LocalSettings.php [hunk 5]
```diff
@@ -2656,6 +2656,13 @@
 		'default' => [ 'png' ],
 	],
 	// Namespaces [MWNamespaces]
+		'wgExtraSignatureNamespaces' => [
+		'default' => [],
```
Comment (paladox) [inline LocalSettings.php]: Please make this:

```
'wgExtraSignatureNamespaces' => [
	'default' => [],
	'nonciclopediawiki' => [
		NS_HELP,
		NS_PROJECT,
	],
],
```

Also move this to some where under '// Namespaces [MWNamespaces]' or in that section please.

**Pull request merged**